Sources/Shared/VimeoClient.swift

```diff
@@ -401,30 +438,38 @@ extension VimeoClient {
     /// Singleton instance for VimeoClient. Applications must call configureSharedClient(withAppConfiguration appConfiguration:)
     /// before it can be accessed.
     public static var sharedClient: VimeoClient {
-        guard let _ = self._sharedClient.configuration,
+        guard
+            let _ = self._sharedClient.configuration,
             let _ = self._sharedClient.sessionManager else {
-            assertionFailure("VimeoClient.sharedClient must be configured before accessing")
-            return self._sharedClient
+                assertionFailure("VimeoClient.sharedClient must be configured before accessing")
+                return self._sharedClient
         }
-        
         return self._sharedClient
     }
-    private static let _sharedClient = VimeoClient(appConfiguration: nil, sessionManager: nil)
+    
+    private static let _sharedClient = VimeoClient()
     
     /// Configures the singleton sharedClient instance. This function allows applications to provide
     /// client specific app configurations at start time.
     ///
     /// - Parameters:
     ///   - appConfiguration: An AppConfiguration instance
     ///   - configureSessionManagerBlock: a block to configure the session manager
-    public static func configureSharedClient(withAppConfiguration appConfiguration: AppConfiguration, configureSessionManagerBlock: ConfigureSessionManagerBlock?) {
+    public static func configureSharedClient(
+        withAppConfiguration appConfiguration: AppConfiguration,
+        configureSessionManagerBlock: ConfigureSessionManagerBlock? = nil,
+        reachabilityMonitor: ReachabilityManagingType? = VimeoReachabilityProvider.reachabilityManager
```

Comment:
Another spot where we could swap parameter orders to allow trailing-closure syntax at the invocation site.